andela/trime
# Make project page responsive

### What does this PR do?

Makes the project page responsive.
### Description of Task to be completed?

Make the project page responsive using the Materialize's grid system
### How should this be manually tested?

Open the profile page. Use Chrome Dev Tools or any other appropriate tools to see how the page appears in different screen sizes.
### What are the relevant pivotal tracker stories?

[Make project page responsive](https://www.pivotaltracker.com/story/show/128310123)
### Screenshots

<img width="1126" alt="screen shot 2016-09-06 at 10 32 07" src="https://cloud.githubusercontent.com/assets/18402277/18265278/86938dfc-741d-11e6-8e23-da1643c39ea9.png">
<img width="1125" alt="screen shot 2016-09-06 at 10 32 37" src="https://cloud.githubusercontent.com/assets/18402277/18265281/86b1cc0e-741d-11e6-8200-f4550a7b4e06.png">
<img width="1127" alt="screen shot 2016-09-06 at 10 32 57" src="https://cloud.githubusercontent.com/assets/18402277/18265279/869904da-741d-11e6-9766-95e2189c857d.png">
<img width="1680" alt="screen shot 2016-09-06 at 10 34 17" src="https://cloud.githubusercontent.com/assets/18402277/18265280/86a48cec-741d-11e6-9a49-3197856c569e.png">
